Chatbot.py

```diff
@@ -428,6 +428,40 @@ async def runAgent(userQuery, chat_history):
     # iconIndex = index % len(listIcons)
     questionBtns.append(st.button(f":large_green_circle: {question}", type="secondary"))
 
+if 'clicked' not in st.session_state:
+    st.session_state.clicked = False
+clickedQuestion = None
+
+if 'clicked_question' not in st.session_state:
+    st.session_state.clicked_question = None
+
+def click_button(question):
+    print("button clicked!" + question)
+    st.session_state.clicked = True
+    st.session_state.clicked_question = question
+
+follow_up_questions = []
+followUpBtns = []
+def suggestFollowUps():
+    follow_up_query = f"""
+    Given this chat history {chat_history}, Suggest 2 follow-up questions the user 
+    might ask next. Give your answer in the format where each line has a separate 
+    question. Do not number your responses."""
+
+    follow_up_suggestions = client.chat.completions.create(
+        messages=[{"role": "user", "content": follow_up_query}],
+        model="gpt-4-turbo-preview",
```

Comment:
Consider pinning to a specific model, e.g., `"gpt-4-turbo-2024-04-09"`.

You might try `"gpt-4o-2024-05-13"`, which is faster and cheaper (but maintains good performance).